src/plugin-slots/AdditionalCourseContentPluginSlot/README.md

```diff
@@ -4,3 +4,73 @@
 
 ### Slot ID Aliases
 * `additional_course_content_plugin`
```

Comment:
Out of scope for this PR, but this is a bad name. Shouldn't it be something like `pages_resources_content_permissions_additional_cards` ?

Reply:
The *alias* can't change for backwards compatibility reasons, but I'm very open to changing the id and making the current id (`org.openedx.frontend.authoring.additional_course_content_plugin.v1`) another alias.